Makefile

```diff
@@ -0,0 +1,19 @@
+all: build
+
+run: build
+	redis-metrics
+
+build: dependencies
+	go install github.com/daime/redis-metrics
+
+dependencies:
+	go get github.com/garyburd/redigo/redis
+
+test:
+	go test
+
+clean:
+	go clean
+
+env:
+	export GOPATH=${GOPATH}:${PWD}
```

Comment:
I didn't get this. Why export PWD if GOPATH must contains the project directory?

https://golang.org/doc/code.html#Organization